docs/batch.rst

```diff
@@ -0,0 +1,122 @@
+################
+Batch operations
+################
+
+.. _batch:
+
+POST /batch
+===========
+
+**Requires an FxA OAuth authentication**
+
+The POST body is a mapping, with the following attributes:
+
+- ``requests``: the list of requests
+- ``defaults``: (*optional*) values in common for all requests
+
+ Each request is a JSON mapping, with the following attribute:
+
+- ``method``: HTTP verb
+- ``path``: URI
+- ``body``: a mapping
+- ``headers``: (*optional*), otherwise take those of batch request
+
+::
+
+    {
```

Comment:
If we provide examples, we should do it with HTTPie so that it's easier for clients to reproduce.